_posts/2023-01-26-jpastreamer-extension.adoc

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: 'Express Hibernate Queries as Type-Safe Java Streams'
+date: 2023-01-26
```

Comment:
Could you update the date and the file name for Thursday? It will then be published automatically on Thursday and I will tweet about it.

Reply:
Should we do Friday instead, considering it is now Thursday?

Reply:
I updated it to 2023-02-10, let me know if I need to change it again.